File: HeartForCharity/HeartForCharity.Services/Database/Country.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HeartForCharity.Services.Database
{
    public class Country
    {
        [Key]
        public int CountryId { get; set; }

        [Required]
        [MaxLength(100)]
        public string Name { get; set; } = null!;

        [MaxLength(10)]
        public string? ISOCode { get; set; }

        public virtual ICollection<City> Cities { get; set; } = new List<City>();
    }
}

File: HeartForCharity/UI/heartforcharity_shared/lib/model/responses/country.dart
class Country {
  final int countryId;
  final String name;
  final String? isoCode;

  Country({this.countryId = 0, this.name = '', this.isoCode});

  factory Country.fromJson(Map<String, dynamic> json) => Country(
        countryId: json['countryId'] ?? 0,
        name: json['name'] ?? '',
        isoCode: json['iSOCode'],
      );
}
